pycbc/live/snr_optimizer.py

```diff
@@ -0,0 +1,427 @@
+# Copyright (C) 2023
```

Comment:
```suggestion
# Copyright (C) 2023 Arthur Tolley
```

Reply:
I did start doing this but didn't know whether to add the names of everyone who has worked on the snr optimization or just my own? Felt weird to just put myself

Reply:
Too late, oh well